Makefile

```diff
@@ -244,3 +244,8 @@ bin/werf: bin bin/trdl ## Install werf for images-tags generator.
 .PHONY: update-k8s-patch-versions
 update-k8s-patch-versions: ## Run update-patchversion script to generate new version_map.yml.
 	cd candi/tools; bash update_kubernetes_patchversions.sh
+
+##@ Lib helm
+.PHONY: update-lib-helm
+update-lib-helm:
+	cd helm_helpers/ && yq -i '.dependencies[0].version = "$(version)"' Chart.yaml && helm dependency update && tar -xf charts/deckhouse_lib_helm-*.tgz -C charts/ && git add Chart.yaml Chart.lock charts/*
```

Comment:
The archive file is not being cleaned. Do we need it committed? I believe we should add gitignore entry for it

> charts/deckhouse_lib_helm-*.tgz

Reply:
added to gitignore